packages/cli/src/CredentialsHelper.ts

```diff
@@ -37,6 +37,7 @@ import { RESPONSE_ERROR_MESSAGES } from './constants';
 import { CredentialsRepository } from '@db/repositories/credentials.repository';
 import { SharedCredentialsRepository } from '@db/repositories/sharedCredentials.repository';
 import { CredentialNotFoundError } from './errors/credential-not-found.error';
+// eslint-disable-next-line n8n-local-rules/misplaced-n8n-typeorm-import
 import { In } from '@n8n/typeorm';
 import { CacheService } from './services/cache/cache.service';
 
```

Comment:
We should move all DB related code out of this file, and into `src/credentials/credentials.service.ts`. we need fewer of these `*Helper` files.